docs/coding-guidelines/code-formatting-tools.md

```diff
@@ -0,0 +1,79 @@
+# Code Formatting Tools in dotnet/runtime
```

Comment:
Maybe link to https://github.com/dotnet/jitutils for JIT developers requiring jit-format?